backend/src/main/java/de/echochamber/backend/controller/PollController.java

```diff
@@ -66,98 +66,51 @@ public PollController(PollService pollsService, UserService userService, AnswerS
                     .chosenAnswer(possibleAnswerEntity)
                     .user(userEntityOptional.get())
                     .build();
-            return ok(map(pollsService.giveAnswer(givenAnswer)));
+            return ok(mapper.map(pollsService.giveAnswer(givenAnswer)));
         }
         throw new EntityNotFoundException("User not found");
     }
 
-    @GetMapping
+    @GetMapping(value ="/all")
```

Comment:
```suggestion
    @GetMapping(value ="/")
```
Get the whole collection via get to the root endpoint